components/sections/SponsorShowcase.tsx

```diff
@@ -13,9 +13,10 @@ const toGridTemplate = (rows: string[]) =>
 const sponsorsWideGrid = toGridTemplate([
   `deloitte deloitte innovapost innovapost solace solace`,
   `cdw cdw kanatanorthbia kanatanorthbia ssc ssc`,
+  `.. .. sunlife sunlife .. ..`,
```

Comment:
Looks great, thank you Zach!

Though I'm just curious, what does `..` do here compared to just `.`? It seems to be working fine either way

Reply:
That was a mistake but yeah it should work either ways.